src/js/controller/instream-adapter.js

```diff
@@ -275,6 +276,7 @@ const InstreamAdapter = function(_controller, _model, _view, _mediaPool) {
             data.tag = _options.tag;
         }
         this.trigger(MEDIA_COMPLETE, data);
+        _model.set('isAdPlaying', false);
```

Comment:
Don't forget outstream adapter. I really think you should be setting this off of api.on('adBreakStart'), and api.on('adComplete') instead to ensure we aren't missing something. Every ad plugin dispatches these events to the player.